### PR TITLE
Function clipper handle file

### PR DESCRIPTION
**Pull request by llopez**

### CODE REVIEW

File: src/mappings/modules/liquidation/clipper.ts [hunk 1]
```diff
@@ -5,9 +5,50 @@ import {
   Take as TakeEvent,
   Yank as YankEvent,
   Redo as RedoEvent,
+  File as FileBigIntEvent,
+  File1 as FileAddressEvent,
```
Comment (rtomas) [inline src/mappings/modules/liquidation/clipper.ts:9]: It would be better to have a more memotecnic name

File: subgraph.yaml [hunk 2]
```diff
@@ -659,9 +659,9 @@ dataSources:
         - SaleAuction
       eventHandlers:
         - event: File(indexed bytes32,uint256)
-          handler: handleFile
+          handler: handleFile1
```
Comment (rtomas) [inline subgraph.yaml:662]: Maybe change the name to handleFileBigInt

File: subgraph.yaml [hunk 3]
```diff
@@ -659,9 +659,9 @@ dataSources:
...
         - event: File(indexed bytes32,address)
-          handler: handleFile
+          handler: handleFile2
```
Comment (rtomas) [inline subgraph.yaml:664]: Maybe change the name to handleFileAddress

File: src/mappings/modules/liquidation/clipper.ts (hunk 1 above)
Comment (llopez) [reply src/mappings/modules/liquidation/clipper.ts:9]: I think the naming is ok here.

File: subgraph.yaml (hunk 2 above)
Comment (llopez) [reply subgraph.yaml:662]: I want to keep the default function handlers.

File: subgraph.yaml (hunk 3 above)
Comment (llopez) [reply subgraph.yaml:664]: default function handle suts better

**Pull request merged**